app/Util/ActivityPub/Inbox.php

```diff
@@ -293,8 +293,6 @@ public function handleDeleteActivity()
         if(!isset(
             $this->payload['actor'], 
             $this->payload['object'], 
```

Comment:
is the comma here misplaced, which makes federation stop working? see pixelfed-dev chat…